api/templates/verification_code_not_given_2nd_email.html

```
@@ -1,374 +1,19 @@
-		-->
-		<!--[if mso 12]>
-			<style type="text/css">
-				.flexibleContainer{display:block !important; width:100% !important;}
-			</style>
-		<![endif]-->
-		<!--[if mso 14]>
-			<style type="text/css">
-				.flexibleContainer{display:block !important; width:100% !important;}
-			</style>
-		<![endif]-->
-	</head>
-	<body style="background-color: #DEDEDE; margin-left: 0; margin-top: 0;">
+{% extends "email.html" %}
+{% load header_image from enrolment_email %}
+{% load external_url from constants_tags %}
 
-		<!-- CENTER THE EMAIL // -->
-		<!--
-		1.  The center tag should normally put all the
-			content in the middle of the email page.
-			I added "table-layout: fixed;" style to force
-			yahoomail which by default put the content left.
-		2.  For hotmail and yahoomail, the contents of
-			the email starts from this center, so we try to
-			apply necessary styling e.g. background-color.
-		-->
-		<center style="background-color:#DEDEDE;">
-			<table border="0" cellpadding="0" cellspacing="0" width="100%" id="bodyTable" style="table-layout: fixed;max-width:100% !important;width: 100% !important;min-width: 100% !important; heigh: 100%;">
-				<tr>
-					<td align="center" valign="top" id="bodyCell">
+{% block heading %}Find a Buyer service{% endblock %}
+{% block to %}{{ full_name|default:"User" }}{% endblock %}
 
-						<!-- EMAIL HEADER // -->
-						<!--
-							The table "emailBody" is the email's container.
-							Its width can be set to 100% for a color band
-							that spans the width of the page.
-						-->
-						<table bgcolor="#DEDEDE" border="0" cellpadding="0" cellspacing="0" width="500" id="emailHeader">
+{% block body %}
+<p>You recently created a profile for your company on the Find a Buyer service, and we sent you a verification code by post 8 days ago.</p>
+<p>You can enter the code at <a href="{{ verfication_url }}" target="_blank">{{ verfication_url|default:"http://great.gov.uk/verify" }}</a>.</p>
```

The .../verify URL was supplied by Alfie in response to my asking what is was. As you will remember the verfication_url was from you. Either way, I'd see the injection of actual correct functionality-based content, through whatever chosen method, to be a BE decision. I have no problem with whatever you decide but would expect a hardcoded value to raise eyebrows.